assets/airports/kpns.json

```diff
@@ -0,0 +1,2678 @@
+            "airspace_class": "C",
+            "poly": [
+                ["N30d013m14.997", "W88d001m29.995"],
+                ["N30d015m03.074", "W87d048m19.429"],
+                ["N30d041m27.104", "W87d042m30.668"],
+                ["N30d040m58.353", "W87d048m28.791"],
+                ["N30d038m00.575", "W87d055m02.307"],
+                ["N30d032m32.916", "W87d054m02.141"]
+            ]
+        },
+        {
+            "floor": 0,
+            "ceiling": 100,
+            "airspace_class": "C",
+            "poly": [
+                ["N030d041m27.104", "W087d042m30.668"],
+                ["N030d057m56.761", "W087d038m53.574"],
+                ["N031d003m03.508", "W087d035m02.016"],
+                ["N031d009m58.480", "W086d059m58.519"],
+                ["N030d059m57.732", "W086d043m56.174"],
+                ["N030d053m20.899", "W086d046m54.223"],
+                ["N030d042m16.736", "W086d046m52.728"],
+                ["N030d038m43.457", "W086d054m55.666"],
+                ["N030d043m58.589", "W086d054m57.007"]
+
+            ]
+        }
+    ],
+    "fixes": {
+        "ALICO": ["N030d030m55.659", "W087d011m57.627"],
+        "BRENT": ["N030d033m39.672", "W087d012m41.462"],
+        "BRATT": ["N030d046m09.880", "W087d040m33.250"],
+        "CDUBB": ["N030d046m50.941", "W087d029m09.985"],
+        "CEMRI": ["N030d030m47.289", "W086d057m59.562"],
+        "CEW"  : ["N030d049m32.610", "W086d040m34.997"],
+        "CILEB": ["N030d039m31.207", "W087d014m15.533"],
+        "DARED": ["N031d000m05.349", "W085d051m02.357"],
+        "ENNTT": ["N031d041m21.413", "W086d019m02.276"],
+        "FIDUL": ["N030d028m58.463", "W087d007m57.631"],
+        "HEGOB": ["N030d025m52.279", "W087d024m50.323"],
+        "HEVVN": ["N029d049m19.113", "W083d053m42.894"],
+        "INBRD": ["N030d048m29.962", "W087d000m27.109"],
+        "ITNEW": ["N030d048m49.168", "W086d054m39.446"],
+        "LGC"  : ["N033d002m56.825", "W085d012m22.400"],
+        "LOXLY": ["N030d045m42.620", "W087d047m56.110"],
+        "MILVE": ["N030d027m51.899", "W087d014m01.223"],
+        "NGS"  : ["N030d036m54.562", "W086d056m14.533"],
+        "NPA"  : ["N030d021m28.595", "W087d018m59.505"],
+        "NSE"  : ["N030d043m26.417", "W087d001m05.010"],
+        "NUN"  : ["N030d028m19.709", "W087d020m09.238"],
+        "OBUFY": ["N030d016m34.510", "W087d008m08.287"],
+        "PEDPE": ["N030d029m29.973", "W087d005m04.898"],
+        "PENSI": ["N030d047m35.683", "W087d016m25.529"],
+        "PIGON": ["N031d033m33.581", "W086d039m51.174"],
+        "ROMEK": ["N031d011m08.429", "W086d040m18.640"],
+        "SJI"  : ["N030d043m33.531", "W088d021m33.448"],
+        "TIGLE": ["N030d027m16.208", "W087d017m15.489"],
+        "VARRE": ["N030d021m41.591", "W086d051m48.911"],
+        "VLKNN": ["N033d040m12.474", "W086d053m58.824"],
+        "WABEN": ["N030d040m12.227", "W087d014m26.464"],
+        "ZUGTI": ["N030d044m42.820", "W088d003m48.735"]
+    },
+    "restricted": [
```

Comment:
The coordinates don't seem quite right

Compare these:
![image](https://user-images.githubusercontent.com/13125311/222142039-e8150fd7-e946-4ecd-8d94-12fcc77801a3.png)
![image](https://user-images.githubusercontent.com/13125311/222142140-36a453c0-67ef-4bc4-afb7-58f480eadd20.png)

W-155 has two arcs on it's north edge which seem to be centered on the NPA VOR (I could be wrong, you may find better information)

The south edge of R-2908 should line up perfectly with W-155

It looks like there's some self intersecting lines in the R-2915 polygon currently. It may be easier to separate it into its different sections

Reply:
@svalencia014 Definitely better, but there's still some issues. 

- The R2908 doesn't line up with the video map at all, feel free to take the coordinates straight from the video map instead.
- The W155 is still missing its arcs. You can use GPSVisualizer to try calculate the coordinates, or, perhaps easier, take the coordinates used in the video map.

Reply:
The initial coordinates were pulled directly from the PNS video maps in the VRC Sector file provided by ZJX, and they supposedly "don't seem right". Do you want me to use the video map or not?

Reply:
If they're straight from the video map, they should line up with the video map. Could be coordinate conversion issue maybe. I'd have to have a look at the original VRC file to see if I can find anything